app/src/main/java/org/ole/planet/myplanet/library/AdapterLibrary.java

```diff
@@ -144,6 +146,11 @@ public ViewHolderLibrary(View itemView) {
             ratingBar = itemView.findViewById(R.id.rating_bar);
             checkBox = itemView.findViewById(R.id.checkbox);
             llRating = itemView.findViewById(R.id.ll_rating);
+            llRating.setVisibility(Constants.showBetaFeature(Constants.KEY_RATING, context) ? View.VISIBLE :View.GONE );
+            average = itemView.findViewById(R.id.average);
+            average.setVisibility(Constants.showBetaFeature(Constants.KEY_RATING, context) ? View.VISIBLE :View.GONE );
+            tv_rating = itemView.findViewById(R.id.tv_rating);
```

Comment:
@inDepthh  There is no TextView with id tv_rating in row_library.xml , that's why you are facing the problem.

Rating number textview has id : rating in xml.
<img width="596" alt="screen shot 2019-01-06 at 7 54 37 pm" src="https://user-images.githubusercontent.com/9145217/50737098-97032d80-11ed-11e9-9d91-1673e51fc052.png">

You don't need to define new variable tv_rating, it is already there i.e rating.
<img width="849" alt="screen shot 2019-01-06 at 7 57 14 pm" src="https://user-images.githubusercontent.com/9145217/50737095-89e63e80-11ed-11e9-9c4b-b1bcd6745f07.png">

Reply:
@rrijal53 Yes, problem was that I was using `tv_rating` from `fragment_library_detail` I was not intending to reference `row_library`. Thanks.